### PR TITLE
Update to d3fc v6.0.0

### PR DESCRIPTION
**Pull request by rjmcneill**
Relates to #650

### CODE REVIEW

File: src/assets/js/chart/primary.js [hunk 1]
```diff
@@ -121,6 +124,7 @@ export default function() {
         case 'line':
         case 'point':
         case 'area':
+            currentSeries.option.xValue(function(d) { return d.date; });
```
Comment (jleft) [inline src/assets/js/chart/primary.js]: Can we not set this when we initialise the models? The function name - `updateYValueAccessorUsed` - is now misleading as it also affects the xValue accessor.

File: src/assets/js/chart/primary.js [hunk 2]
```diff
@@ -109,7 +109,10 @@ export default function() {
 
     function updateMultiSeries() {
         var baseChart = [gridlines, currentSeries.option, closeLine];
-        var indicators = currentIndicators.map(function(indicator) { return indicator.option; });
+        var indicators = currentIndicators.map(function(indicator) {
```
Comment (jleft) [inline src/assets/js/chart/primary.js]: Can we not set this when we initialise the models? Rather than every time we update the multi?

**Comment by rjmcneill** [comment]
@jleft is this more what you had in mind?

**Comment by jleft** [comment]
> @jleft is this more what you had in mind?

Yeah, exactly - thanks! :+1:

**Pull request merged**